test/fixtures/es-module-loaders/builtin-named-exports-loader.mjs

```diff
@@ -28,7 +28,7 @@ export function load(url, context, next) {
       source: generateBuiltinModule(urlObj.pathname),
       format: 'commonjs',
     };
-  } else if (context.format === 'commonjs') {
+  } else if (context.format === undefined || context.format === 'commonjs') {
```

Comment:
did not try this locally but would it work to have the `context.format` set to the exact input value returned also on the `shortCircuit` path? Or is there a reason to return `commonjs` also if it was `undefined` on input?
```
    return {
      shortCircuit: true,
      source: readFileSync(new URL(url)),
      format: context.format,
    };
```

Reply:
I don’t understand this note. Could you please explain?

Reply:
before the change, the resolution was to short-circuit and set `format` to `commonjs` if it was `commonjs`. Now we set `commonjs` if it was either  `commonjs` or `undefined`. The question would be if it would be enough to return whatever was there (`context.format`). Not sure if the test still passes with that change.

Reply:
> would it work to have the `context.format` set to the exact input value returned also on the `shortCircuit` path?

[`format` is a required property on the return value from user-specified `load` hooks](https://nodejs.org/api/module.html#loadurl-context-nextload), and it can’t be undefined. Therefore we can’t return `context.format`, since `context.format` might be undefined.

There might be some existing hooks out in the wild that make an invalid assumption that `context.format` is always defined, but they’re already mistaken per current docs. After this PR lands there will be cases where there was previously a `context.format` hint but now there no longer is, and that might be a frustrating change to hook authors but I would argue that their code was buggy beforehand.